aci/resource_aci_rest_managed.go

```diff
@@ -269,15 +269,17 @@ func MakeAciRestManagedQuery(d *schema.ResourceData, m interface{}, method strin
 	if method == "POST" {
 		content := d.Get("content")
 		contentStrMap := toStrMap(content.(map[string]interface{}))
+		contentStrMap["annotation"] = "orchestrator:terraform"
 
 		childrenSet := make([]interface{}, 0, 1)
 
 		for _, child := range d.Get("child").(*schema.Set).List() {
 			childMap := make(map[string]interface{})
 			childClassName := child.(map[string]interface{})["class_name"]
-			childContent := child.(map[string]interface{})["content"]
+			childContent := toStrMap(child.(map[string]interface{})["content"].(map[string]interface{}))
+			childContent["annotation"] = "orchestrator:terraform"
```

Comment:
I changed the order so that the static `orchestrator:terraform` annotation is added to the contents first and can then be overwritten if a user supplies an annotation value in the `content` of a `child`